File: backend/micro_epoweredmove_reviews_reservations/src/main/java/com/kosdiam/epoweredmove/models/enums/TeslaCompatible.java
package com.kosdiam.epoweredmove.models.enums;

public enum TeslaCompatible {
    YES("Yes"),
    NO("No");

    private String teslaCompatible;
    private TeslaCompatible(String teslaCompatible){
        this.teslaCompatible = teslaCompatible;
    }

    @Override
    public String toString(){
        return teslaCompatible;
    }
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/enums/TeslaCompatible.kt
package com.kosdiam.epoweredmove.models.enums

import com.kosdiam.epoweredmove.R
import com.kosdiam.epoweredmove.models.interfaces.IEnumName

enum class TeslaCompatible (override val stringResource: Int) : IEnumName {
    YES(R.string.tesla_compatible_yes),
    NO(R.string.tesla_compatible_no);
}